docs/deployment/troubleshooting.md

```diff
@@ -20,6 +20,10 @@ You can tune the values for the number of tasks each queue worker may run in par
 
 The [OpenFaaS workshop](https://github.com/openfaas/workshop) has more instructions on running tasks asynchronously.
 
+* Debugging
```

Comment:
(Suggested verbiage)

* Verbose output.

The Queue Worker component enables asynchronous processing of function requests. The default verbosity level hides the message content, but this can be viewed by setting `write_debug` to `true` when deploying.